docker-compose.yml

```diff
@@ -38,6 +38,14 @@ services:
       - ~/.aws:/root/.aws:ro  # map credentials to be used by boto3, read-only
     command: ["poetry", "run", "performance_manager"]
 
+  test_mssql:
+    container_name: test_mssql
+    env_file: .env
+    build: .
+    working_dir: /lamp
+    network_mode: "host"
+    command: ["poetry", "run", "test_mssql"]
+
```

Comment:
I will drop this before the final merge of this pull request into main, but this was helpful for me locally in confirming all of this works.

Reply:
i had to add a block to avoid DNS errors.

```
    extra_hosts:
      - "HSTMDM01:172.29.21.44"
 ```